tests/unit/App.spec.ts

```diff
@@ -603,15 +609,23 @@ describe('App component - other', () => {
     // GET org info
     get.withArgs('https://auth.api.url/orgs/668')
       .returns(Promise.resolve({
-        data: {}
+        data: {
+          mailingAddress: {
+            city: 'Victoria',
+            country: 'CA',
+            postalCode: 'V8W 3E6',
+            region: 'BC',
+            street: '2-940 Blanshard St',
+            streetAdditional: ''
+          }
+        }
```

Comment:
Hah, so this was an example of an org with missing mailing address (which I asked about earlier).

Therefore, having the data above is needed, but I think the conditional chaining isn't needed (or desired).

I guess my approach is I'd rather see an error than a hidden fallback value.